libs/server-sdk/src/evaluation/operators.cpp

```diff
@@ -89,9 +89,6 @@ bool RegexMatch(std::string const& context_value,
         // boost::bad_expression can be thrown by basic_regex when compiling a
         // regular expression.
         return false;
-    } catch (boost::regex_error) {
```

Comment:
Turns out `boost::bad_expression` (above the fold) is actually a typedef for `boost::regex_error`.